plugins/native_dio_adapter/lib/src/conversion_layer_adapter.dart

```diff
@@ -34,10 +35,12 @@ class ConversionLayerAdapter implements HttpClientAdapter {
   Future<BaseRequest> _fromOptionsAndStream(
     RequestOptions options,
     Stream<Uint8List>? requestStream,
+    Future<dynamic>? cancelFuture,
   ) async {
-    final request = Request(
+    final request = AbortableRequest(
       options.method,
       options.uri,
+      abortTrigger: cancelFuture,
```

Comment:
Could you add tests regarding the aborted behavior in `test/conversion_layer_adapter_test.dart`?